tests/queries/0_stateless/02403_enable_date32_results.sql.j2

```diff
@@ -0,0 +1,8 @@
+{% for option_value in ['true', 'false'] -%}
```

Comment:
Looking at the file name, I wonder what are .j2 tests and how can they be executed?

Reply:
> Looking at the file name, I wonder what are .j2 tests and how can they be executed?

tests/clickhouse-test handles jinja2 template tests transparently, no extra care is required.

Reply:
@rschu1ze I attach script.sql file. That is what j2 test generates during its work. There are some j2 tests in CH tests folder. So j2 support is enabled by default for tests in CH. 
[script.txt](https://github.com/ClickHouse/ClickHouse/files/9572553/script.txt)